project/api/management/commands/sync_TEs.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand, CommandError
+from api.models import TerritorialEntity,SpacetimeVolume
+
+class Command(BaseCommand):
+    help = 'For syncing TR inception/dissolution dates with STVs we have.'
```

Comment:
Well, in case we are using existing STVs it might be better to do it as a calculated value.
This is a part of the #186 correct?

cc @whirish

Reply:
Yes, this would definitely be better suited to a post-save hook for STVs

Reply:
That is a short SQL query to update existing TEs
```sql
UPDATE api_territorialentity
SET inception_date = min_date, dissolution_date = max_date
FROM (
	SELECT entity_id, MIN(start_date) AS min_date, MAX(end_date) AS max_date FROM api_spacetimevolume
	GROUP BY (entity_id)
-- 	ORDER BY entity_id
) AS foo
WHERE  api_territorialentity.id = foo.entity_id;
```